File: app/src/at-nhatnguyen/java/asiantech/internship/summer/javacore/Exercise8.java
package asiantech.internship.summer.javacore;
public class Exercise8 {
        private static boolean isReversibleNumber(int n){
            StringBuilder x = new StringBuilder();
            String str= ""+n;
            x.append(str);
            String check= ""+x.reverse();
            return str.equals(check);
        }
        public static void main(String[] args) {
            int n,count=0;
            for(n=100000 ; n<= 999999 ; n++){
                if(isReversibleNumber(n)){
                    System.out.println(n);count++;
                }
            }
            System.out.println("Co "+count+" so thuan nghich co 6 chu so");
        }
    }

File: app/src/at-nhatnguyen/java/asiantech/internship/summer/kotlin/Exercise8.kt
package asiantech.internship.summer.kotlin

private fun isReversibleNumber(n: Int): Boolean {
    val x = StringBuilder()
    val str =  n.toString()
    x.append(str)
    val check =  x.reverse().toString()
    return str == check
}

fun main() {
    var n = 100000
    var count = 0
    while (n <= 999999) {
        if (isReversibleNumber(n)) {
            println(n)
            count++
        }
        n++
    }
    print("Co $count so thuan nghich co 6 chu so")
}
